src/main/java/com/googleinterns/zoomtube/servlets/LectureServlet.java

```diff
@@ -48,11 +45,21 @@
       + "watch\\?v%3D|watch\\?feature=player_embedded&v=|%2Fvideos%2F|"
       + "embed%\u200C\u200B2F|youtu.be%2F|%2Fv%2F)[^#\\&\\?\\n]*";
 
+  @VisibleForTesting static final String PARAM_ID = "id";
+
   /* Name of input field used for lecture name in lecture selection page. */
-  private static final String NAME_INPUT = "name-input";
+  @VisibleForTesting static final String PARAM_NAME = "name-input";
 
   /* Name of input field used for lecture video link in lecture selection page. */
-  private static final String LINK_INPUT = "link-input";
+  @VisibleForTesting static final String PARAM_LINK = "link-input";
+
```

Comment:
nit: you do not need 2 new lines. 1 is fine.

Reply:
Removed